src/main/java/org/jabref/JabRefException.java

```diff
@@ -3,11 +3,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// Class for handling exceptions in GUI and printing corresponding exception messages
 public class JabRefException extends Exception {
-
+    
+    // Creating a logger and using it to write messages to log files
     private static final Logger LOGGER = LoggerFactory.getLogger(JabRefException.class);
     private String localizedMessage;
 
+    // Passing the exception message to the base class
```

Comment:
Not a Javadoc comment. Obvious purpose of the constructor as this is the usual pattern for exception classes in Java.

Reply:
Okay.
I understand this. I will try to fix the said issues as soon as possible. 

Thank you.